src/borg/remote.py

```diff
@@ -78,6 +126,22 @@ def __init__(self, restrict_to_paths, append_only):
         self.repository = None
         self.restrict_to_paths = restrict_to_paths
         self.append_only = append_only
+        self.client_version = (1, 1, 0)
```

Comment:
This one actually is the last borg version that used the old protocol. This is duplicated once in the RemoteRepository code so this still needs to be improved. But it‘s not something we need to maintain. But i should add a comment.